docs/fflags.adoc

```diff
@@ -0,0 +1,23 @@
+== Feature Flags
+
+The following system properties are available for toggling or specifying particular features to turn on or off.
+Like any system properties, they should be considered experimental and subject to removal without notice.
```

Comment:
Perhaps link to https://www.jenkins.io/doc/book/managing/system-properties/ for usage etc.?